### PR TITLE
Tramp Freighter remap

### PR DESCRIPTION
**Pull request by hazelrat**
This is a full remap for the Tramp Freighter offship that intends to bring the ship into closer alignment with newer mapping standards, while playing the same role as the original map. 

- Although the old Catspaw-class freighter is nostalgic, I don't think the layout is really tenable now without changing it so much that it might as well have been a full remap anyways. The tramp freighter plays a very important role as the one universal offship - every whitelist, every sector, any reason you can RP up for why you're doing anything you're doing.
- The layout is very different, primarily to accommodate full docking arms. The idea is that the freighter should always be able to move cargo to and from other parties without having to traipse through space. With this design, even if the other party doesn't have docking ports, they can still just go over to the tramp freighter to conduct their business in person. You can even just cater to people if you like, it has a hydroponics bay and bartending equipment.
- All references to the ship that show up on IC mediums have been renamed to 'Independent Freighter'.
- The cargo bay of the ship has been expanded. Four crates in the middle of the room contain particularly valuable randomised items. The shuttle is also designed to accommodate loading racks with merchandise like you would on the merchant's shuttle.
- The guns in the armoury are half static, half random. The static options are an energy rifle, energy pistol, and antique revolver - so you'll always have *some* decent options.
- The ship generally is *full* of contraband, even outside of the smuggling compartments. Don't get pulled over by a patrol ship, and if you are, take a lot of care to clean up shop. 
- The ship is deliberately weak in ship combat. It has relatively little fuel and very punishing entry points - you can hit the bridge from any direction but the aft, and doing so also cuts the gunnery. In my view, the tramp should be the 'small fry' in ship combat, any pirate offship should have a relatively easy time with it.

### CODE REVIEW

**Comment by hazelrat** [comment]
!wip

**Comment by hazelrat** [comment]
!review

**Comment by DreamySkrell** [comment]
image
the shuttle overmap object should be on the holopad, just to be consistent with most other maps, and possibly more easily noticed when mapping

**Comment by DreamySkrell** [comment]
image
this is a merchant shuttle with very little storage space for crates or the like

image
I'd widen it a bit like this, provide some free tiles for storing anything in the middle, and behind the passenger seats

**Comment by DreamySkrell** [comment]
imageimage
do not change the layer var of things

**Comment by DreamySkrell** [comment]
image
new docking/shuttle airlocks should be defined in code, not in the map
see scc scout or sol yacht for an example
external access airlocks are fine to be defined in the map

**Comment by DreamySkrell** [comment]
image
do not use fake objects
add a indoor lattice like that into code, like `/obj/structure/lattice/catwalk/indoor/lattice`
but also I am really not a fan of a decorative lattice, that look exactly the same as actual functional lattices

**Comment by DreamySkrell** [comment]
image
these docks are not complaint with the dock/shuttle standards, cannot fit max size shuttles
this is a merchant ship so it should be fully compliant in this aspect

**Comment by DreamySkrell** [comment]
image
this is the antag gun spawner, that can spawn crazy powerful stuff including like pulse rifles
replace with civ gun spawners or something else, in smaller number too, cause they already do have a armory on the ship

**Comment by DreamySkrell** [comment]
image
replace with `/obj/machinery/iff_beacon` to allow them to change both designation and class (ICV/etc)

**Comment by DreamySkrell** [comment]
image
do not have to change it here, but in future maps use large tank markers, example on scc scout or sol yacht

**Comment by DreamySkrell** [comment]
image
this is radioactive goo that irradiates whoever tries to clean it
replace it with vomit or something

**Comment by DreamySkrell** [comment]
image
eepy

**Comment by DreamySkrell** [comment]
image
glass door to the armory?

File: maps/away/ships/tramp_freighter/tramp_freighter.dm [hunk 1]
```diff
@@ -63,34 +69,60 @@
 	return skybox_image
 
 /obj/effect/shuttle_landmark/tramp_freighter/nav1
```
Comment (DreamySkrell) [inline maps/away/ships/tramp_freighter/tramp_freighter.dm:71]: Split all the landmarks into their own file like `tramp_freighter_landmarks.dm`

**Comment by DreamySkrell** [comment]
image
mining mecha here? should it not be a loader to load the ship gun?

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
.

**Comment by hazelrat** [comment]
Implemented all changes, plus an addition of disposals and some smaller tweaks.

**Comment by DreamySkrell** [comment]
image
still see a lot of objects with changed layers on the map

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**